docs/admin-ports.md

```diff
@@ -13,6 +13,11 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 62053: dcos-spartan
  - 62080: dcos-navstar
 
+#### Disterl ports:
```

Comment:
These are already in the list above, and this is user-facing / visible, where people won't know / be interested in "disterl ports" vs. "general ports"